test_suite/tests/test_annotations.rs

```diff
@@ -977,3 +977,83 @@ fn test_invalid_length_enum() {
         Error::Message("invalid length 1, expected tuple of 2 elements".to_owned()),
     );
 }
+
+impl Into<Option<u32>> for EnumToU32 {
```

Comment:
This would be more readable if you declare the enum before starting to declare impls on it.